src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -784,7 +784,7 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 	async openEditors(editors: Array<IEditorInputWithOptions | IResourceEditorInputType>, group?: OpenInEditorGroup): Promise<IEditorPane[]> {
 
 		// Convert to typed editors and options
-		const typedEditors: IEditorInputWithOptions[] = editors.map(editor => {
+		let typedEditors: IEditorInputWithOptions[] = editors.map(editor => {
```

Comment:
Why does this only apply to `openEditors` what about `openEditor` method?

Reply:
openEditors is the only one that currently acts on side by side groups since the explorer is the only place where we currently allow open to the side and when you open to the side you can open multiple